apps/frontend/src/hooks/useEvents.ts

```diff
@@ -19,6 +21,8 @@ export const useItems = () => {
         setItems(response.data)
       } catch (error) {
         console.error("failed to fetch items: ", error)
```

Comment:
Not sure but would user feedback for API errors be good?

Reply:
added for debuggin in development will remove it